client/python/README.md

```diff
@@ -1,3 +0,0 @@
-This folder contains ModelDB's
```

Comment:
Did you mean to delete these? Those links might have relevant info, please double check.